tests/end_to_end/conftest.py

```diff
@@ -57,7 +57,13 @@ def configure_environment(request):
     # For each configuration playbook previously declared in the test, get the complete path and run it
     for playbook in getattr(request.module, 'configuration_playbooks'):
         configuration_playbook_path = os.path.join(getattr(request.module, 'test_data_path'), 'playbooks', playbook)
-        ansible_runner.run(playbook=configuration_playbook_path, inventory=inventory_playbook)
+        parameters = {'playbook': configuration_playbook_path, 'inventory': inventory_playbook}
+
+        # Check if the module has extra variables to pass to the playbook
+        configuration_extra_vars = getattr(request.module, 'configuration_extra_vars', None)
+        parameters.update({'extravars': configuration_extra_vars})
```

Comment:
```suggestion
        if configuration_extra_vars is not None:
          parameters.update({'extravars': configuration_extra_vars})
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/3012/commits/6e00ee2439db6b1ffb77c8d7befafac18f869528